packages/block-library/src/search/index.php

```diff
@@ -157,6 +157,31 @@ function classnames_for_block_core_search( $attributes ) {
 	return implode( ' ', $classnames );
 }
 
+/**
+ * Generates a border radius inline style for the wrapper element. It will
+ * adjust the supplied radius value to account for the default padding, making
+ * it visually consistent with the border radii applied to inner elements.
+ * `calc()` is used to support non-pixel CSS units.
+ *
+ * @param string $css_property The CSS property for the inline style.
+ * @param string $value        The border radius value to apply to the property.
+ *
+ * @return string A border radius inline style.
+ */
+function get_wrapper_radius_style( $css_property, $value ) {
```

Comment:
Because this file doesn't use namespaces, should we use a more specific function name to avoid a potential collision with a plugin or other file using this name? Like the above function, maybe we could append `_for_block_core_search` to the function name?

```suggestion
function get_wrapper_radius_style_for_block_core_search( $css_property, $value ) {
```

Reply:
Thanks for catching that oversight! 

I've just pushed a commit renaming it. The suggested suffix has been added and I've removed the `get_` prefix as well just to shorten it a little. It's also a little more inline with the `classnames_for_block_core_search` function as well.